code/modules/media/jukebox.dm

```diff
@@ -997,6 +997,27 @@ var/global/list/loopModeNames=list(
 /obj/machinery/media/jukebox/superjuke/adminbus/cultify()
 	return
 
+obj/machinery/media/jukebox/holyjuke
+	name = "Holyjuke"
+	desc = "The Pastor's jukebox. You feel a weight being lifted simply by basking in its presence."
+
+	state_base = "holyjuke"
+	icon_state = "holyjuke"
+
+	change_cost = 0
+
+	playlist_id="holy"
+	// Must be defined on your server.
+	playlists=list(
+		"holy" = "Pastor's Paradise"
+	)
+
+/obj/machinery/media/jukebox/holyjuke/attackby(obj/item/W, mob/user)
+	// EMAG DOES NOTHING
+	if(istype(W, /obj/item/weapon/card/emag))
+		to_chat(user, "<span class='warning'>A guiltiness fills your heart as a higher power pushes away [W]!</span>")
```

Comment:
\the [W]

Reply:
\the would give me two thes. 
![Yep](https://user-images.githubusercontent.com/52895053/115125942-34369a00-9f91-11eb-96c2-0bedb8c8f53d.png)

Reply:
the fuck

Reply:
It wouldn't. Manually typing `\the` doesn't do fucking anything because it's automatically inserted if it's not there. We just do it for style. So yes, you should add it.